src/index.html

```diff
@@ -22,6 +22,624 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="header__link"
+      >
+        <img
+          class="header__logo"
+          src="images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <a
+        class="header__menu"
+        href="#menu"
+      >
+        <img
+          src="images/menu.png"
+          alt="menu"
+        />
+      </a>
+
+      <nav class="nav header__nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <aside
+      class="menu"
+      id="menu"
+    >
+      <div class="menu__top">
+        <a
+          href="#"
+          class="menu__link"
+        >
+          <img
+            class="menu__logo"
+            src="images/logo.png"
+            alt="logo"
+          />
+        </a>
+        <a
+          class="menu__icon"
+          href="#"
+        >
+          <img
+            src="images/icon-close.png"
+            alt="icon-close"
+          />
+        </a>
+      </div>
+      <div class="nav menu__nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Samsung
+            </a>
+          </li>
```

Comment:
Remember the empty lines between multiline sibling elements [Reference](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules#:~:text=Add%20empty%20lines%20between%20multiline%20sibling%20blocks%20of%20HTML)